### PR TITLE
{ACR} Use azure.multiapi.storage for storage blob service

### PR DESCRIPTION
**Pull request by Juliehzl**
**Description**  

Because the latest `azure.storage.blob` is for track2 without supporting for `BlockBlobService`, `AppendBlobService` ..., which are in track1, using `azure.multiapi.storage` can prevent the breaking change in `azure.stroage.blob`.

**Testing Guide**  

From my investigation, these changes will influence `az acr build` and `az acr pack build`, but there is no test for these commands.
@yugangw-msft  Can you help test them? And I will recommend you to add test for such commands. 

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S169

**Comment by Juliehzl** [comment]
@yugangw-msft  Tested with your commands and there is no problem.

File: src/azure-cli/azure/cli/command_modules/acr/_archive_utils.py [hunk 1]
```diff
@@ -54,6 +54,7 @@ def upload_source_code(client,
         raise CLIError("Failed to get a SAS URL to upload context.")
 
     account_name, endpoint_suffix, container_name, blob_name, sas_token = get_blob_info(upload_url)
+    BlockBlobService = get_sdk(cmd.cli_ctx, ResourceType.DATA_STORAGE, 'blob#BlockBlobService')
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/acr/_archive_utils.py:57]: Can we import it statically like `from azure.storage.blob import BlockBlobService`?
What is the advantage of this approach?

Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/acr/_archive_utils.py:57]: as what I described in description part

**Comment by nwipfli** [comment]
It seems that there are still compatibility issues with 2.6.0:

➜  ~ az group create \
    --name $RESOURCE_GROUP \
    --location $REGION_NAME
/usr/local/Cellar/azure-cli/2.6.0/libexec/lib/python3.8/site-packages/jmespath/visitor.py:32: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if x is 0 or x is 1:
/usr/local/Cellar/azure-cli/2.6.0/libexec/lib/python3.8/site-packages/jmespath/visitor.py:32: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if x is 0 or x is 1:
/usr/local/Cellar/azure-cli/2.6.0/libexec/lib/python3.8/site-packages/jmespath/visitor.py:34: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif y is 0 or y is 1:
/usr/local/Cellar/azure-cli/2.6.0/libexec/lib/python3.8/site-packages/jmespath/visitor.py:34: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif y is 0 or y is 1:
/usr/local/Cellar/azure-cli/2.6.0/libexec/lib/python3.8/site-packages/jmespath/visitor.py:260: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if original_result is 0:
{
  "id": "/subscriptions/a481f10b-726d-493f-930f-ab1bee4b539a/resourceGroups/rg-nwi-aksworkshop",
  "location": "northeurope",
  "managedBy": null,
  "name": "rg-nwi-aksworkshop",
  "properties": {
    "provisioningState": "Succeeded"
  },

**Comment by Juliehzl** [comment]
@nwipfli Thanks for your feedback. We will look into it and @haroldrandom for compatibility issue.

**Comment by fengzhou-msft** [comment]
@nwipfli the issue you reported has been fixed in https://github.com/Azure/azure-cli/pull/13435 and will be released on June 2nd.

**Pull request merged**